docs/AboutUs.md

```diff
@@ -48,12 +48,12 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 * Role: Developer
 * Responsibilities: Dev Ops + Threading
 
-### James Doe
+### Vinod s/o Jaya Kumar
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/Vinod.png" width="200px">
```

Comment:
I think the image and portfolio file name should be your GitHub username in all lowercase as mentioned [here](https://nus-cs2103-ay2223s1.github.io/website/schedule/week7/project.html#2-update-project-website-aboutus-readme)

Reply:
Thank you for catching this